src/main/java/kuromi/command/MarkCommand.java

```diff
@@ -0,0 +1,39 @@
+package kuromi.command;
+
+import kuromi.Storage;
+import kuromi.Ui;
+import kuromi.task.Task;
+import kuromi.task.TaskList;
+
+/**
+ * Mark task as done.
+ */
+public class MarkCommand extends Command {
+    /** Index of the task to be marked **/
+    private int idx;
+
+    /**
+     * Main constructor.
+     *
+     * @param idx Index of the task to be marked.
+     */
+    public MarkCommand(int idx) {
+        this.idx = idx;
+    }
+
+    /**
+     * Mark the task as done in the task list.
+     * Ask UI to print output.
+     *
+     * @param tasks Task list.
+     * @param ui UI of the application to interact with users.
+     * @param storage Storage to update when there is an update with the task list.
+     */
```

Comment:
Good job on a highly descriptive javadocs that can help readers understand your code more easily!